src/ray/raylet/node_manager.cc

```diff
@@ -2216,12 +2222,17 @@ void NodeManager::SendSpilledObjectRestorationRequestToRemoteNode(
           entry->second.first, entry->second.second, client_call_manager_));
   raylet_client->RestoreSpilledObject(
       object_id, spilled_url, node_id,
-      [](const ray::Status &status, const rpc::RestoreSpilledObjectReply &r) {
+      [object_id, node_id, callback = std::move(callback)](
+          const ray::Status &status, const rpc::RestoreSpilledObjectReply &r) {
         if (!status.ok()) {
-          RAY_LOG(WARNING) << "Failed to send a spilled object restoration request to a "
-                              "remote node. This request will be retried. Error message: "
+          RAY_LOG(WARNING) << "Failed to send a spilled object restoration request for "
+                           << object_id << " to a remote node " << node_id
+                           << ". This request will be retried. Error message: "
                            << status.ToString();
         }
```

Comment:
I don't think we need this log warn since we are passing the status to the callback, consider removing it.